Proofreading - previously checked files (#11750)
openvinotoolkit/openvino: docs/install_guides/installing-openvino-apt.md

### Step 1: Set Up the OpenVINO Toolkit APT Repository

> 1. Install the GPG key for the repository.

Author: ```suggestion
1. Install the GPG key for the repository:
```

### Step 4 (Optional): Install Software Dependencies

> Once you have installed OpenVINO runtime and also decide to [install OpenVINO Model Development Tools](installing-model-dev-tools.md), make sure that you install external software dependencies first. 

Author: ```suggestion
Once you have installed OpenVINO runtime and also decided to install [OpenVINO Model Development Tools](installing-model-dev-tools.md), make sure that you install external software dependencies first. 
```

> Refer to <a href="openvino_docs_install_guides_installing_openvino_linux.html#install-external-dependencies">Install External Software Dependencies</a> for detailed steps.

Author: ```suggestion
For detailed information, refer to the <a href="openvino_docs_install_guides_installing_openvino_linux.html#install-external-dependencies">Install External Software Dependencies</a>.
```

.. tab:: NCS 2

   To perform inference on Intel® Neural Compute Stick 2 powered by the Intel® Movidius™ Myriad™ X VPU, follow the steps in :ref:`NCS2 Setup Guide <ncs guide>`.

>    <!--For more details, see the `Get Started page for Intel® Neural Compute Stick 2 <https://software.intel.com/en-us/neural-compute-stick/get-started>`.-->

Author: ```suggestion
   For more details, see the [Get Started page for Intel® Neural Compute Stick 2](https://software.intel.com/en-us/neural-compute-stick/get-started).
```
